<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculationResultDlg</class>
 <widget class="QDialog" name="CalculationResultDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Convergence Summary</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="3">
    <widget class="QLineEdit" name="targetDeltaR">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Target Delta-R</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="avDeltaR">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Average Delta-R for last iteration</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="iterationCount">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of Iterations</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableWidget" name="vesselTable">
     <column>
      <property name="text">
       <string>Vessel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delta-R</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font: bold</string>
     </property>
     <property name="text">
      <string>Largest Delta-R by Vessel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>iterationCount</tabstop>
  <tabstop>avDeltaR</tabstop>
  <tabstop>targetDeltaR</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
